client/src/components/LevelLayout.tsx

```diff
@@ -50,6 +56,19 @@ const LevelLayout: React.FC<Props> = ({
         {showInput && <StepValidation />}
         {children}
       </Container>
+      <Flex>
+        <Spacer />
+        <Box
+          display="flex"
+          alignItems="center"
+          bg="gray.200"
+          p={4}
+          borderWidth="1px"
+          borderRadius="lg"
+        >
+          {showAttempts && <Text>Remaining Attempts: {game.attempts} /3 </Text>}
```

Comment:
super duper nit: please add a whitespace between `/` and `3`

Reply:
Or remove the whitespace between `{game.attempts}` and `/3` so the result is something like: `1/3`

Reply:
Resolved in last push.